docs/design-proposals/apiserver/openstack-api-support.md

```diff
@@ -0,0 +1,408 @@
+---
+title: Support Openstack Compute APIs in Arktos
+authors:
+  - "@yb01"
+---
+
+# Support Openstack Compute APIs in Arktos
+## Background and context
+To boost Arktos VM adoption and to let Openstack users to easily explore Arktos VM support without a lot of changes from the users’ applications, it is decided to add a new set of APIs in Arktos to match Openstack Server related operations. Besides better user experiences, this new APIs will also help existing Openstack perf tests to be relatively easily converted to run against Arktos.
+
+## Goal and non-goals
+
+Goals:
```

Comment:
You can specify release cycle for goal and non goal. Otherwise, non goal could be an future goal.

Reply:
Add for Arktos 130 release